lib/modules/manager/asdf/extract.spec.ts

```diff
@@ -486,13 +504,27 @@ dummy 1.2.3
             packageName: 'ocaml/ocaml',
             depName: 'ocaml',
           },
+          {
+            currentValue: '3.50.0',
+            datasource: 'github-releases',
+            packageName: 'oracle/oci-cli',
+            depName: 'oci',
+            extractVersion: '^(?<version>\\S+)',
```

Comment:
```suggestion
            extractVersion: '^v(?<version>\\S+)',
```

git tags have leading `v` and renovate will use release tag instead of title.

Reply:
corrected